data_models/migrations/0024_populate_show_digas_id.py

```diff
@@ -31,13 +39,24 @@ def populate_digas_id(apps, schema_editor):
                 continue
 
             # We found something!
-            show.digas_id = digas_id_by_on_demand_link[sod_url]
+            digas_id = digas_id_by_on_demand_link[sod_url]
+            if digas_id in used_digas_ids: 
+                # Alert the user and don't update this show.
+                log.exception('Duplicate digas_id encountered for {show}.' + \
+                    'digas_id not updated.'.format(
+                        show=show.name,
+                    )
+                )
+                break
+
+            show.digas_id = digas_id
             show.save()
+            used_digas_ids.add(digas_id)
             # Break out of this inner for loop, continuing with next show
             break
         else:
             # We never found a satisfying episode, warn user about this
-            print(
+            log.exception(
```

Comment:
`log.error` seems more appropriate. `log.exception` should only be called from an exception handler, per https://docs.python.org/3.6/library/logging.html#logging.Logger.exception since it is intended to add information from the exception currently being handled.
```suggestion
            log.error(
```